### ISSUE TITLE
WebP Does Not Respect EXIF Orientation

### ISSUE THREAD

**Issue by Mafus1** [Jeavon/Slimsy#73]
Images that have been rotated using the EXIF attribute `Orientation`  are displayed in the wrong orientation when converted to a WebP.
In my opinion, the EXIF attribute `Orientation`  should be considered when the library converts the image to a WebP. Especially since browsers generally respect this attribute when displaying JPGs or other image types.

**Comment by alangilby** [comment]
I'm also facing this issue. Thanks for all the work on this plugin, it's superb.

**Comment by Jeavon** [comment]
ImageSharp.Web does have the [autoorient ](https://github.com/SixLabors/ImageSharp.Web/pull/260)command. Could you try adding `&autoorient=true` to your image as a test?

We could consider adding a global option to Slimsy or the tag helper to always add this command...

I would interested in testing this with Cloudflare processing also if you were able to share a test image with this issue.

**Comment by alangilby** [comment]
https://leftlion.co.uk/media/ohnham1g/image0.jpeg is an image that has this issue (i've turned off webp for now so they are appearing correctly). I'm using the tag helper for these images so `<slimsy-picture media-item="@typedMediaPickerSingle" width="@mediaItem.UmbracoWidth" height="@mediaItem.UmbracoHeight" css-class="block  max-w-full w-full  mx-auto" render-lqip="true" render-webp-alternative="false"></slimsy-picture>` Which means no eays way to edit the url.

**Comment by Jeavon** [comment]
https://leftlion.co.uk/media/ohnham1g/image0.jpeg?width=180&height=135&format=webp&quality=70&autoorient=true
Woohoo, it works, let's add this option

**Comment by Jeavon** [comment]
@alangilby which version are you using? v4.2 or v5.1?

**Comment by Jeavon** [comment]
Latest 5.1.1 pre-release has this option
e.g.
```
 "Slimsy": {
   "WidthStep": 180,
   "UseCropAsSrc": false,
   "DefaultQuality": 70,
   "AppendSourceDimensions": true,
   "AutoOrient": true,
   "TagHelper": {
     "SingleSources": [ "gif" ],
     "DefaultPictureSources": [
       {
         "Extension": "webp",
         "Quality": 90
       },
       {
         "Extension": "avif",
         "Quality": 100
       }
     ],
     "ImageDimensions": true
   }
 }
```

`Install-Package Our.Umbraco.Slimsy -Version 5.1.1-alpha.519 -Source https://www.myget.org/F/umbraco-packages/api/v3/index.json`

**Comment by alangilby** [comment]
Amazing. I'm on 5.1, will install the pre-release now. Much appreciated

**Comment by alangilby** [comment]
I'm currently getting a not found error:
 Install-Package Our.Umbraco.Slimsy -Version 5.1.1-alpha.519 -Source https://www.myget.org/F/umbraco-packages/api/v3/index.json
Install-Package : Unable to find package source https://www.myget.org/F/umbraco-packages/api/v3/index.json. Use Get-PackageSource to see all available package sources.
At line:1 char:1
+ Install-Package Our.Umbraco.Slimsy -Version 5.1.1-alpha.519 -Source h ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package], Exception
    + FullyQualifiedErrorId : SourceNotFound,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage

**Comment by Jeavon** [comment]
Odd, it is there, maybe try dotnet instead

`dotnet add package Our.Umbraco.Slimsy --version 5.1.1-alpha.519 --source https://www.myget.org/F/umbraco-packages/api/v3/index.json`

**Comment by Jeavon** [comment]
https://www.myget.org/feed/umbraco-packages/package/nuget/Our.Umbraco.Slimsy/5.1.1-alpha.519

**Comment by alangilby** [comment]
Worked fine with dotnet (might be me using VSCode). And all working as expected. thanks for the swift update, very  much appreciated.

**Comment by Jeavon** [comment]
Great! Will get this released

**Comment by Jeavon** [comment]
Released in v5.1.1 - thanks @alangilby
